### PR TITLE
Discussions status badges

### PR DESCRIPTION
**Pull request by jstefaniuk-d2l**
The status badge logic now allows for the discussions set of statuses

image

### CODE REVIEW

**Comment by jstefaniuk-d2l** [comment]
Note to self: this can be simplified once chris's PR goes in

**Comment by ChrisKraljevicD2L** [comment]
Code LGTM, I can test for discussions and regressions in quizzes/assignments

File: behaviors/types-behavior.html [hunk 1]
```diff
@@ -22,7 +23,8 @@
 							instructionsRel: this.HypermediaRels.Assignments.instructions,
 							userActivityUsageClass: this.HypermediaClasses.activities.userAssignmentActivity,
 							activityRel: this.HypermediaRels.assignment,
-							activityClass: this.HypermediaClasses.assignments.assignment
+							activityClass: this.HypermediaClasses.assignments.assignment,
+							noCompletion: false
```
Comment (CodeBaboon) [inline behaviors/types-behavior.html]: This property goes away if the suggestion [here](https://github.com/Brightspace/d2l-activity-details-ui/pull/87#discussion_r206997573) is accepted.

Comment (jstefaniuk-d2l) [reply behaviors/types-behavior.html]: This repo does things differently than the other repo, so doing an override wouldn't work at the moment

**Pull request merged**